enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,982 @@
+    message: LoadBalancer is not progressing
+    reason: LoadBalancerNotProgressing
+    status: "False"
+    type: LoadBalancerProgressing
+...
+...
+  - lastTransitionTime: "2024-04-17T03:09:32Z"
+    message: |-
+      One or more status conditions indicate unavailable: LoadBalancerReady=False 
+      (SyncLoadBalancerFailed: The service-controller component is reporting 
+      SyncLoadBalancerFailed events like: Error syncing load balancer: failed to ensure
+      load balancer: error creating load balancer: "ResourceInUse: The allocation IDs 
+      are not available for use\n\tstatus code: 400, request id: 58136c42-e1bd-4c1a-ac38-7078da2b1d95"
+      The kube-controller-manager logs may contain more details.)
+    reason: IngressControllerUnavailable
+    status: "False"
+    type: Available
+
+```
+
+3. Are there any pre-requisites for this EP ?
+  Yes, please refer section `Pre-requisites` under [Workflow Description](#Workflow Description)
+
+4. What happens when the cluster-admin specified invalid EIPs in install-config.yaml at installation time (day 0) ?
+  We would try to get the installer feasibly validate the install config for day 0. However, we still need verify this
+  with the installer team. Mostly, we will need to add validations in [pkg/types/aws/validation/platform.go](https://github.com/openshift/installer/blob/master/pkg/types/aws/validation/platform.go)
+  Things to check with the installer team:
+  - How to find the number of public subnets for the VPC(which will be created by the installer)? Shall
+  we check compare the eips with the number of Availability Zones ?
+  - When VPC is already created and we already know the VPC ID can we check the number of subnets using the
+  same logic present in the [e2e tests](https://github.com/openshift/cluster-ingress-operator/pull/1034/commits/fba43803e232277a96565b1eb8875d19df9bf66e#diff-160d46f77f47e8e0d35737b019c27a6e4a6f7e7eae4f6e85c0dcdb43b20a2379R163).
+  Related comments:
+  - [discussion on separate the validation for BYOB subnets and installer-created subnets](https://github.com/openshift/enhancements/pull/1593#discussion_r1593385372)
+  - [workflow for when the installer creates the VPC and public subnets](https://github.com/openshift/enhancements/pull/1593#discussion_r1594164039)
+
+  To summarize shall we proceed with the following ?(Installer team to confirm):
+  - For validation for BYOB subnets:
+  The logic as per [e2e tests](https://github.com/openshift/cluster-ingress-operator/pull/1034/commits/fba43803e232277a96565b1eb8875d19df9bf66e#diff-160d46f77f47e8e0d35737b019c27a6e4a6f7e7eae4f6e85c0dcdb43b20a2379R163) will work 
+  if added to [pkg/types/aws/validation/platform.go](https://github.com/openshift/installer/blob/master/pkg/types/aws/validation/platform.go). Is the installer team OK with that ?
+
+  - For validation for  Installer Created Subnets:
+  number of EIPs must match number of AZs in region (installer team please confirm this).  
```

Comment:
Yes this seems reasonable. Users can specify a subset of AZs in the installconfig, but it should still be relatively straightforward.

Reply:
OK thanks for confirming!